src/main/java/ua/lviv/iot/parkingServer/logic/services/interfaces/VehicleServiceInterface.kt

```diff
@@ -0,0 +1,6 @@
+package ua.lviv.iot.parkingServer.logic.services.interfaces
+
+import ua.lviv.iot.parkingServer.model.Vehicle
+
+interface VehicleServiceInterface : GeneralServiceInterface<Vehicle, Long> {
+}
```

Comment:
Remove brackets because your interface has empty body. Add newline at the end of file